gapic-generator/templates/default/service/client/method/def/_deprecated.erb

```diff
@@ -0,0 +1,5 @@
+<%- assert_locals method -%>
+<%- if method.is_deprecated? -%>
+  warn "[DEPRECATION]: <%= method.name %> is deprecated."
```

Comment:
For libraries to print deprecation warnings to stderr is considered annoying in the Ruby community. I would rather we didn't do it.

If a decision has been made to do this uniformly across all languages, then at the least I'd like the following  done:

* Add `if $VERBOSE` to this line so the warning is printed only in verbose-mode
* Provide more context in the warning message. As it is, it's not very actionable because it doesn't tell which class the method is on, nor who is calling it.

Reply:
Removed the message printing.